src_Core/ISA/ISA_Decls_Priv_S.bsv

```diff
@@ -409,17 +418,19 @@ function Tuple2#(Bool,Exc_Code) is_pte_fault
 
    Bool access_cap_ok = (   (! capability)
 			 || access_load
-			 || (access_store && (pte_StoreCap == 1'b1)));
+			 || (access_store && (pte_StoreCap == 1'b1));
 
    Bool pte_a_d_fault = (   fn_PTE_to_A (pte) == 0)
 			 || ((! read_not_write) && (fn_PTE_to_D (pte) == 0));
 
+   Bool pte_cd_fault = capability && !read_not_write && pte_CapDirty == 1'b0;
```

Comment:
```suggestion
   Bool pte_cd_fault = (capability && (! read_not_write) && (pte_CapDirty == 1'b0));
```
would be the right style, or if you want to wrap the long-ish line (looks weird here because mixing tabs and spaces interacts poorly with diff markers)
```suggestion
   Bool pte_cd_fault = (   capability
			&& (! read_not_write)
			&& (pte_CapDirty == 1'b0));
```
(`set sw=3 sts=3 ts=8 noet` for indentation, and the first line always leaves a space where the operator would be)